### ISSUE TITLE
Env - Configurable corrections on Temperature

### ISSUE THREAD

**Issue by fel115** [ElektraInitiative/opensesame#70]
@markus2330 what is meant by this task? Which correction? Something like a closed-loop controller?

**Comment by markus2330** [comment]
It is simply a correction value to be added/removed to adjust/calibrate the temperature. The user only should see the corrected value.

**Comment by fel115** [comment]
So the configurable correction could be `-0.4°C` and the sensor temp `26.8°C`, then should the user see `26.4°C` as a result?

**Comment by markus2330** [comment]
Exactly. This is useful as the MOD-ENV are quite inaccurate. I didn't check MOD-IR yet (compare with high-precision thermometer) but probably something similar is useful there, too.

**Comment by fel115** [comment]
According to the [data-sheet](https://www.olimex.com/Products/Modules/Sensors/MOD-IR-TEMP/resources/MLX90614_rev001.pdf) the MOD-IR has a high accuracy. `High accuracy of 0.5°C over wide temperature
range (0..+50°C for both Ta and To)`.

Should I create a new PR for this issue, after #63 is merged?

**Comment by markus2330** [comment]
Not high-prior, let us do the refactoring first.

**Comment by markus2330** [comment]
Not high-prior, let us do the refactoring first.

**Comment by markus2330** [comment]
@fel115 do you already have a fix for this?

**Comment by fel115** [comment]
Not implemented now
